src/modules/utility.js

```diff
@@ -0,0 +1,47 @@
+const getItems = () => {
+  let items;
+  if (localStorage.getItem('items') === null) {
+    items = [];
+  } else {
+    items = JSON.parse(localStorage.getItem('items'));
+  }
+  return items;
+};
+
+const addToLocalStorage = (item) => {
+  const items = getItems();
+  items.push(item);
+  localStorage.setItem('items', JSON.stringify(items));
+};
+
+const updateItemInLocalStorage = (index, updatedItem) => {
+  const todoList = getItems();
+  const itemIndex = todoList.findIndex((item) => item.index === index);
+  console.log(itemIndex);
+  if (itemIndex !== -1) {
+    todoList[itemIndex] = updatedItem;
+    localStorage.setItem('items', JSON.stringify(todoList));
+  }
+};
```

Comment:
- [ ] when I edit a task, it does not update the local storage, kindly fix that
![image](https://github.com/ClarenceAtim/ToDoList/assets/66097371/b9b9b4af-3f2e-485a-8fdd-35f14ec820f1)